docs/admin/kubernetes-ops/admin-ui-private.md

```
@@ -85,23 +85,23 @@ This way the calls are made privately without hitting the FQDN through the inter
                   # add a new argument to load self-signed cert
                   - --default-ssl-certificate=<namespace>/internal-tls-certificate
         ```
-        
+
     1.  Rollout restart the `ingress-nginx-controller` deployment.
 
 1.  Expose the service IP (port 443) to host (port 8443):
 
     ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)